packages/core/src/ui/app.tsx

```diff
@@ -76,13 +77,14 @@ export const buildUiState = () => {
     },
 
     port: 0,
+    hostname: "localhost",
```

Comment:
Made a change so that if the user has not specified a custom hostname, it will display `localhost` instead of `0.0.0.0` (matches Next.js behavior).

The key fix (display the custom hostname if the user provided one) remains. Does that seem OK @jaylmiller ?

Reply:
`0.0.0.0` seems more accurate since thats what we default to, no?